3rdparty/dlmalloc/dlmalloc/malloc.h

```diff
@@ -1,12 +1,12 @@
 /*
```

Comment:
Is there any modification to this file other than formatting? If not, I would prefer to keep the file as it is. I believe this file was directly copied from the 3rd party source. Please correct me if I'm wrong. Thanks!

Reply:
Sure, me too. But the clang-format hook disagrees. So this won't commit without override, nor pass in the CI.

Maybe 3rdparty should be exempt from formatting? That's what we do in CCF. I do not feel strongly about this, I'm happy to go with anything.

Reply:
Yes you are correct --- 3rdparty should be exempt from formatting. However, we currently seems to do that manually.

In this case you need to do force push (I remember there is a option to ignore format checks on certain files).

Reply:
I have done that now.